flink-table/flink-table-planner-blink/src/main/scala/org/apache/flink/table/planner/calcite/FlinkPlannerImpl.scala

```diff
@@ -142,10 +142,14 @@ class FlinkPlannerImpl(
         return sqlNode
       }
       sqlNode match {
+        // this explain should be deprecated
```

Comment:
Explain will not be deprecated... It is the `ExplainOperation` to be deprecated.